api/src/main/java/org/openmrs/module/pihcore/task/PihRemovePatientsFromMCOEQueue.java

```diff
@@ -0,0 +1,106 @@
+package org.openmrs.module.pihcore.task;
+
+import org.apache.commons.lang.time.StopWatch;
+import org.openmrs.Concept;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.queue.api.QueueServicesWrapper;
+import org.openmrs.module.queue.api.search.QueueEntrySearchCriteria;
+import org.openmrs.module.queue.api.search.QueueSearchCriteria;
+import org.openmrs.module.queue.model.Queue;
+import org.openmrs.module.queue.model.QueueEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+public class PihRemovePatientsFromMCOEQueue implements Runnable {
+
+    private static final Long QUEUE_TIMEOUT_MINUTES = 720L;
```

Comment:
If the ticket description is to end queues > 12 hours, why don't we configure this in hours rather than in minutes?  I understand that minutes gives more granular flexibility, but hours is more easy to follow and read, if that's what the actual requirement is.  I don't see us changing this in the future to a fraction of an hour...